mediapicker/source-camera/build.gradle

```diff
@@ -31,6 +31,15 @@ dependencies {
     implementation "androidx.core:core-ktx:$androidxCoreVersion"
 }
 
+dependencyAnalysis {
+    issues {
+        onUnusedDependencies {
+            // This dependency is actually needed because of 'androidx.core.content.FileProvider' on 'AndroidManifest.xml'.
+            exclude("androidx.core:core-ktx")
```

Comment:
It's quite unexpected that this module has `FileProvider` in the manifest, although it doesn't really use it. I wonder - does it make sense to move decleration of `FileProvider` to manifest of `mediapicker`?

Reply:
Yea, maybe @wzieba , let me take a closer look on in next week and see if that's a better solution, thanks for taking a closer look yourself, much appreciated! 🙇

Reply:
👋 @wzieba !

So, I tried to move the `FileProvider` configuration from the `mediapicker:source-camera` module and into the `mediapicker` module, but then I started to get the below `IllegalArgumentException` exception when trying to open the `CAMERA` using the `SampleApp`.

<details>
    <summary>java.lang.IllegalArgumentException: Couldn't find meta-data for provider with authority org.wordpress.android.sampleapp.provider</summary>

```stacktrace
12:53:09.411  E  FATAL EXCEPTION: main
                 Process: org.wordpress.android.sampleapp, PID: 15307
                 java.lang.IllegalArgumentException: Couldn't find meta-data for provider with authority org.wordpress.android.sampleapp.provider
                 	at androidx.core.content.FileProvider.getFileProviderPathsMetaData(FileProvider.java:688)
                 	at androidx.core.content.FileProvider.parsePathStrategy(FileProvider.java:719)
                 	at androidx.core.content.FileProvider.getPathStrategy(FileProvider.java:669)
                 	at androidx.core.content.FileProvider.getUriForFile(FileProvider.java:451)
                 	at org.wordpress.android.mediapicker.MediaPickerUtils.createCaptureImageIntent(MediaPickerUtils.kt:96)
                 	at org.wordpress.android.mediapicker.ui.MediaPickerFragment.onActionSelected(MediaPickerFragment.kt:293)
                 	at org.wordpress.android.mediapicker.ui.MediaPickerFragment.access$onActionSelected(MediaPickerFragment.kt:75)
                 	at org.wordpress.android.mediapicker.ui.MediaPickerFragment$onViewCreated$3$3.invoke(MediaPickerFragment.kt:259)
                 	at org.wordpress.android.mediapicker.ui.MediaPickerFragment$onViewCreated$3$3.invoke(MediaPickerFragment.kt:231)
                 	at org.wordpress.android.mediapicker.viewmodel.EventKt$observeEvent$1.invoke(Event.kt:36)
                 	at org.wordpress.android.mediapicker.viewmodel.EventKt$observeEvent$1.invoke(Event.kt:36)
                 	at org.wordpress.android.mediapicker.viewmodel.EventKt$sam$androidx_lifecycle_Observer$0.onChanged(Unknown Source:2)
                 	at androidx.lifecycle.LiveData.considerNotify(LiveData.java:133)
                 	at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:151)
                 	at androidx.lifecycle.LiveData.setValue(LiveData.java:309)
                 	at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
                 	at androidx.lifecycle.LiveData$1.run(LiveData.java:93)
                 	at android.os.Handler.handleCallback(Handler.java:959)
                 	at android.os.Handler.dispatchMessage(Handler.java:100)
                 	at android.os.Looper.loopOnce(Looper.java:232)
                 	at android.os.Looper.loop(Looper.java:317)
                 	at android.app.ActivityThread.main(ActivityThread.java:8592)
                 	at java.lang.reflect.Method.invoke(Native Method)
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
```

</details>

I then timeboxed to try and quickly make it work but I couldn't figure this out and then didn't want to spend more time on it unnecessarily. 🤷 

Maybe it is better to keep that configuration there after all, wdyt? 🤔 

For more info see: [MediaPickerUtils.createCaptureImageIntent(...)](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/blob/0af39deb42b3c7879751680ce70ef74986fc49c0/mediapicker/src/main/java/org/wordpress/android/mediapicker/MediaPickerUtils.kt#L80-L105)

Reply:
Have you maybe compared the merged manifests before and after your change? Is there any diff there? If you consider this not important or low priority, I'm also okay with merging as it is, hence the ✅!

Reply:
I actually didn't @wzieba , but I really didn't want to go that far to figure this out, I also didn't want to start playing with Manifest permission and moving them around too, I did try debugging to see whether and how the `authority` is changing but couldn't figure it out, also tried a custom `FileProvider`... 🤷 

I am not sure about how important this is or its priority, but I do consider that out of scope of unused dependencies, especially since the fix doesn't seem to be straightforward and could mean applying more changing that initially expected.

As such, let's merge this as is for now, but I created this #84 issue for us to pick this up at some point in the future (if need be).